runtests.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
Can you please move this `runtests.py` file into the `tests` directory?  I try to avoid having too many files at the root of the repo.